### PR TITLE
Database interface

### PR DESCRIPTION
**Pull request by datacrystals**
@bkl4114 Are you able to go through and fill out the remaining database functions in the DB Interface file? For the Neuron Info function, I suppose I really meant a set of functions there, so when you get there, please just implement what makes sense in your mind.

### CODE REVIEW

**Comment by bkl4114** [comment]
Yep, I'll take a look

**Comment by datacrystals** [comment]
Awesome, thanks.

**Comment by datacrystals** [comment]
Oh just thought I should add that we need a region column in the neuron table for the DB to be able to specify the region that each neuron is in.

**Comment by bkl4114** [comment]
I can only see the first line that you write when I get an email notification of new comments here, so I already added a 'region' column to the neuron table :-)
If we need to add a region table that is easy enough. What are the two sets of coordinates for?

**Comment by datacrystals** [comment]
I thought that might be easier as it would allow us to define a 3d region where all items within that bounding box are part of that region.

File: System/Database/DatabaseInterface.py [hunk 1]
```diff
@@ -58,19 +58,46 @@ def CheckIfUserExists(self, UserName:str): # Checks If A Given User Exists #
 
         # Execute Some MYSQL Cursor Code #
         # something here...
+        
+       
+        #Count of number of Users having the same UserName in the table
+        cnt= self.DatabaseCursor.execute("SELECT * from "+table+" where UserName='"+UserName+"'")
```
Comment (bkl4114) [inline System/Database/DatabaseInterface.py]: If you use this it should just return a number:
"SELECT count(*) from..."

Comment (PRISHIta123) [reply System/Database/DatabaseInterface.py]: Yes, the execute function returns the number of rows that are affected by the query.

**Review by bkl4114 — APPROVED** [review]

**Pull request merged**